### PR TITLE
EZP-27113: Add edit button to sub-item list

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-27113

## Description
Adds the possibility to edit a content from the sub item list. If a draft is present, the draft conflict screen will be displayed.

Screencast: https://drive.google.com/file/d/0B5amepf6EkXbeUpQNVltZk9hQm8/view

## Tests
Manual test and updated unit test

### CODE REVIEW

File: Tests/js/views/ez-searchlistview.html [hunk 1]
```diff
@@ -60,13 +60,17 @@
                 fullpath: "../../../../Resources/public/js/extensions/ez-loadmorepagination.js"
             },
             "ez-subitemlistitemview": {
-                requires: ['ez-templatebasedview', 'ez-translateproperty'],
+                requires: ['ez-templatebasedview', 'ez-translateproperty', 'ez-draftconflict'],
```
Comment (dpobel) [inline Tests/js/views/ez-searchlistview.html]: why this change ?

Comment (yannickroger) [inline Tests/js/views/ez-searchlistview.html]: It won't load the "ez-subitemlistitemview"  without the dependency on the extension (added in this PR) and the test will fail.

Comment (dpobel) [inline Tests/js/views/ez-searchlistview.html]: ah I misread the previous line :) Need more coffee syndrome it seems ;)
